.travis.yml

```diff
@@ -56,8 +57,11 @@ install:
     pip install --upgrade pip
     pip install --upgrade wheel
     pip install --upgrade setuptools
+    if [[ -n ${CYTHON} ]]; then
```

Comment:
Hmm, is this related to this PR?

Reply:
Yes it is. In some of the travis builds cartopy was installed before cython, which broke the build (e.g. https://travis-ci.org/yt-project/yt/jobs/422847999 for f38232307ff43c5605b88b578bb55012ce48f8ee ); this addressed that issue.